beets/autotag/__init__.py

```diff
@@ -45,13 +45,13 @@ def apply_item_metadata(item, track_info):
     if track_info.data_source:
         item.data_source = track_info.data_source
 
-    if track_info.lyricist is not None:
+    if track_info.get('lyricist'):
```

Comment:
to be perfectly honest, I find these tests `if track_info.XX is not None:` extremely ugly, that's why I removed them.